default/stringtables/en.txt

```diff
@@ -12198,26 +12198,21 @@ PLC_CONFORMANCE
 Conformance
 
 PLC_CONFORMANCE_DESC
-'''• Reduces the impact of species likes and dislikes on planet stability due to buildings and policies.
-• Descreases [[metertype METER_TARGET_RESEARCH]] by [[value PLC_CONFORMANCE_RESEARCH_PENALTY_FLAT]] on all planets 
-• Increases [[metertype METER_TARGET_HAPPINESS]] by [[value PLC_CONFORMANCE_STABILITY_BONUS_FLAT]]
-
-On planets with a stability of at least [[value PLC_CONFORMANCE_MIN_STABILITY]]:
-• Increases [[metertype METER_TARGET_CONSTRUCTION]] by [[value PLC_CONFORMANCE_INFRASTRUCTURE_BONUS_FLAT]]
-• Increases [[metertype METER_STOCKPILE]] by [[value PLC_CONFORMANCE_STOCKPILING_BONUS_FLAT]]'''
+'''• Reduces the impact of species dislikes on planet stability due to buildings, policies or focus.
+• Increases [[metertype METER_TARGET_HAPPINESS]] by [[value PLC_CONFORMANCE_STABILITY_BONUS_FLAT]]'''
 
 PLC_CONFORMANCE_SHORT_DESC
-Reduces species opinion effects
+Reduces species dislikes effects and increases stability
 
 PLC_LIBERTY
 Liberty
 
 PLC_LIBERTY_DESC
-'''• Increases the impact of species likes and dislikes on planet stability due to buildings and policies.
-• Increases [[metertype METER_TARGET_RESEARCH]] by [[value PLC_LIBERTY_RESEARCH_BONUS_FLAT]] and decreases [[metertype METER_TARGET_HAPPINESS]] by [[value PLC_LIBERTY_STABILITY_PENALTY_FLAT]] on all planets with a stability of at least [[value PLC_LIBERTY_MIN_STABILITY]].'''
+'''• Doubles the impact of species dislikes on planet stability due to buildings, policies or planetary focus.
+• Increases [[metertype METER_TARGET_RESEARCH]] by 1 RP per each [[value PLC_LIBERTY_RESEARCH_BONUS_DENOMINATOR]] points of [[metertype METER_TARGET_HAPPINESS]] above [[value PLC_LIBERTY_MIN_STABILITY]] on all planets regardless of focus.'''
```

Comment:
the 1 should still be a named value lookup

ideally also "Doubles" but that's a bit awkward to set up

Reply:
I'll rework that part.